src/main/java/com/meilisearch/sdk/Client.java

```diff
@@ -42,10 +42,10 @@ public Index createIndex(String uid) throws Exception {
 	 * Create index
 	 * Refer https://docs.meilisearch.com/references/indexes.html#create-an-index
 	 *
-	 * @param uid        Unique identifier for the index to create
+	 * @param uid Unique identifier for the index to create
 	 * @param primaryKey The primary key of the documents in that index
 	 * @return Meilisearch API response
-	 * @throws Exception If an error occurs
+	 * @throws Exception if the JSON cannot be parsed correctly
```

Comment:
What you wrote is true, but it is just a part of it. Imagine you try to create an Index that already exists. This wouldn't be a JSON parsing problem, yet an Exception will be thrown! So in this case I think it may be better to leave something more generic

Reply:
Looking at it the first time I though the Exception was only being thrown from the `gson.fromJson()`. Didn't think about the call to `indexesHandler.getAll()`. Thanks